src/classes/i18n.js

```diff
@@ -195,6 +195,8 @@ export default class i18n {
              * Since we have a valid updated meta.json, we'll assume the
              * previous download was successful
              */
+            await new Promise(resolve => this.load_files(resolve));
```

Comment:
We don't need to loop through the languages?
What exactly is expected as the return?

On L224
```
this.load_files.bind(this, resolve)
```

Reply:
https://github.com/anyTV/anytv-i18n/blob/3ba16bfe4e7e05551b5a04d7f30d39dcc0703ac4/src/classes/i18n.js#L163-L170

The code uses `this` to identify the path that needs to be loaded. On the callback is passed.